src/main/java/com/azure/cosmos/examples/common/Family.java

```diff
@@ -63,12 +63,12 @@ public void setRegistered(boolean isRegistered) {
         this.isRegistered = isRegistered;
     }
 
-    private String id;
-    private String lastName;
-    private String district;
-    private Parent[] parents;
-    private Child[] children;
-    private Address address;
-    private boolean isRegistered;
+    private String id="";
```

Comment:
code style:
please try to format the changes, spacing, etc
Intellij has support for code style formatting
https://moduscreate.com/blog/12-intellij-idea-keyboard-shortcuts/